### ISSUE TITLE
code-signing-identity and team-id not working

### ISSUE THREAD

**Issue by lolevsky** [yukiarrr/ios-build-action#40]
This is my configuration:
```
uses: yukiarrr/ios-build-action@v1.3.1
   with:
      project-path: ios/my_project.xcodeproj
      workspace-path: ios/my_project.xcworkspace
      team-id: ${{ secrets.TEAM_ID }}
      p12-base64: ${{ secrets.P12_BASE64 }}
      mobileprovision-base64: ${{ secrets.AD_HOC_MOBILEPROVISION_BASE64 }}
      code-signing-identity: ${{ secrets.AD_HOC_CODE_SIGNING_IDENTITY }}
      export-method: ad-hoc
      output-path: output/output.ipa
```

I was not able to sign it was giving me error:
`error: Signing for "my_project" requires a development team. Select a development team in the Signing & Capabilities editor that matches the selected profile "my provisioning". (in target 'my_project' from project 'my_project')`

After I added DEVELOPMENT_TEAM to the project.pbxproj it started to show new error message:
`error: No signing certificate "iOS Development" found: No "iOS Development" signing certificate matching team ID "***" with a private key was found. (in target 'my_project' from project 'my_project')`

Last thing that I did is to add CODE_SIGN_IDENTITY in the project.pbxproj and only then I was able to do successful build.

Would like to understand why it was not able to update it automatically from build step config, should I commit to the repository selected provision and team?

**Comment by yukiarrr** [comment]
Sorry for the late response.
Thanks for the report.
Do you have any other logs of when the error occurred?

**Comment by lolevsky** [comment]
No other logs, only this

**Comment by yukiarrr** [comment]
Thanks for the reply!
In the current information, I don't know the cause of the error.
To investigate this, can you try the following action at the time of the first error?

```yaml
uses: yukiarrr/ios-build-action@v1.3.1
   with:
      project-path: ios/my_project.xcodeproj
      workspace-path: ios/my_project.xcworkspace
      team-id: ${{ secrets.TEAM_ID }}
      p12-base64: ${{ secrets.P12_BASE64 }}
      mobileprovision-base64: ${{ secrets.AD_HOC_MOBILEPROVISION_BASE64 }}
      code-signing-identity: ${{ secrets.AD_HOC_CODE_SIGNING_IDENTITY }}
      export-method: ad-hoc
      output-path: output/output.ipa
      update-targets: | # New!! Targets to be updated with mobileprovision, code signing identity, etc. Split on new lines.
        MyApp
        YourApp
```

**Comment by lolevsky** [comment]
Added but still getting error
```
+-----------------------+------------------------------------+
|             Summary for code signing settings              |
+-----------------------+------------------------------------+
| use_automatic_signing | false                              |
| path                  | ios/react_native_actions.xcodeproj |
| targets               | ["react_native_actions"]           |
+-----------------------+------------------------------------+

+-----------------------------------------------------------+-----------------------------------------------------------+
|                                                Summary for gym 2.170.0                                                |
+-----------------------------------------------------------+-----------------------------------------------------------+
| workspace                                                 | ios/react_native_actions.xcworkspace                      |
| configuration                                             | Release                                                   |
| scheme                                                    | react_native_actions                                      |
| output_directory                                          | output                                                    |
| output_name                                               | output                                                    |
| clean                                                     | true                                                      |
| export_method                                             | ad-hoc                                                    |
| export_options.provisioningProfiles.com.honeybook.demoapp | DemoApp provisioning                                      |
| skip_profile_detection                                    | false                                                     |
| destination                                               | generic/platform=iOS                                      |
| build_path                                                | /Users/runner/Library/Developer/Xcode/Archives/2021-01-03 |
| silent                                                    | false                                                     |
| skip_package_ipa                                          | false                                                     |
| skip_package_pkg                                          | false                                                     |
| result_bundle                                             | false                                                     |
| buildlog_path                                             | ~/Library/Logs/gym                                        |
| use_system_scm                                            | false                                                     |
| xcode_path                                                | /Applications/Xcode_12.2.app                              |
+-----------------------------------------------------------+-----------------------------------------------------------+

❌  error: No signing certificate "iOS Development" found: No "iOS Development" signing certificate matching team ID "***" with a private key was found. (in target 'react_native_actions' from project 'react_native_actions')
```

This is my config:
```
project-path: ios/react_native_actions.xcodeproj
workspace-path: ios/react_native_actions.xcworkspace
team-id: ${{ secrets.TEAM_ID }}
p12-base64: ${{ secrets.P12_BASE64 }}
mobileprovision-base64: ${{ secrets.AD_HOC_MOBILEPROVISION_BASE64 }}
code-signing-identity: ${{ secrets.AD_HOC_CODE_SIGNING_IDENTITY }}
export-method: ad-hoc
output-path: output/output.ipa
update-targets: "react_native_actions"
scheme: react_native_actions
```

**Comment by yukiarrr** [comment]
Thanks!
https://github.com/lolevsky/react_native_actions/runs/1638404039?check_suite_focus=true
I can't view the above link because I don't have permission, so please grant me permission or post the all log.

**Comment by lolevsky** [comment]
@yukiarrr invite sent, let me know if you have issue to access

**Comment by yukiarrr** [comment]
Thanks for the invite!
The bug has been identified and fixed in #47 and #48.
It has been released as 1.3.2, please check it out.

**Comment by lolevsky** [comment]
Thanks @yukiarrr working perfectly!
